parse/lex.go

```diff
@@ -465,14 +469,20 @@ func lexName(l *lexer) stateFn {
 		r rune   // current rune
 	)
 
-	for r = l.next(); isName(r); r = l.next() {
+	for r = l.next(); isName(r) || r == '+'; r = l.next() {
```

Comment:
is `+` not allowed in name?

Reply:
If it is not... does it not conflict with lexName() intent?

Reply:
starting char is: ALPHA / "_" / %xC0--> up with some exclusions
name char is: starting char / DIGIT / "-" / "." / %xB7 and something above 0x300
`+` is 0x2B so doesn't fit in any of previous allowed ranges

Reply:
Does it imply we have to rename `lexName` to `lexNameOrNumber`?

Reply:
Also. If `s` contains `+` and is NOT `number`, it is NOT `name`. Additional check required.